app/views/devise/registrations/new.html.erb

```diff
@@ -1,25 +1,51 @@
-<h2>Sign up</h2>
-
-<%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
-  <%= f.error_notification %>
-
-  <div class="form-inputs">
-    <%= f.input :email,
-                required: true,
-                autofocus: true,
-                input_html: { autocomplete: "email" }%>
-    <%= f.input :password,
-                required: true,
-                hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length),
-                input_html: { autocomplete: "new-password" } %>
-    <%= f.input :password_confirmation,
-                required: true,
-                input_html: { autocomplete: "new-password" } %>
-  </div>
-
-  <div class="form-actions">
-    <%= f.button :submit, "Sign up" %>
-  </div>
-<% end %>
-
-<%= render "devise/shared/links" %>
+<div class="container m-5">
+  <h2>Sign up</h2>
+  
+  <%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= f.error_notification %>
+  
+    <div class="form-inputs">
+      <%= f.input :username,
+                  required: true,
+                  autofocus: true,
+                  input_html: { autocomplete: "username" }%>
+      <%= f.input :email,
+                  required: true,
+                  autofocus: true,
+                  input_html: { autocomplete: "email" }%>
+      <%= f.input :nickname,
+                  required: true,
+                  autofocus: true,
+                  input_html: { autocomplete: "nickname" }%>
+      <%= f.input :first_name,
+                  required: true,
+                  autofocus: true,
+                  input_html: { autocomplete: "first name" }%>
+      <%= f.input :middle_name,
+                  required: true,
+                  autofocus: true,
+                  input_html: { autocomplete: "middle name" }%>
```

Comment:
what does `autocomplete` do here?

Reply:
The `autocomplete` attribute in a `simple_form_for` input field tells the browser whether to automatically suggest previously entered values when the user starts typing in the field (so you should have it everywhere where you have data that you have sued before) 💬

Reply:
BUT I found out that the names should not have spaces but dashes - Added it [here](https://github.com/emmvs/call_your_mom/blob/user/app/views/devise/registrations/new.html.erb#L20-L43)